assets/js/components/ClusterDetails/AscsErsClusterDetails.jsx

```diff
@@ -2,27 +2,78 @@ import React, { useState, useEffect } from 'react';
 
 import PageHeader from '@components/PageHeader';
 import BackButton from '@components/BackButton';
-
+import Table from '@components/Table';
 import ListView from '@components/ListView';
 import ProviderLabel from '@components/ProviderLabel';
 import DottedPagination from '@components/DottedPagination';
+import HostLink from '@components/HostLink';
 
 import ChecksComingSoon from '@static/checks-coming-soon.svg';
 
 import SBDDetails from './SBDDetails';
+import SiteDetails from './SiteDetails';
 import StoppedResources from './StoppedResources';
 
+const nodeDetailsConfig = {
+  usePadding: false,
+  columns: [
+    {
+      title: 'Hostname',
+      key: '',
+      render: (_, { id, name }) => <HostLink hostId={id}>{name}</HostLink>,
+    },
+    {
+      title: 'Role',
+      key: 'roles',
+      render: (content) =>
+        content?.map((role) => role.toUpperCase()).join(', '),
+    },
+    {
+      title: 'Virtual IP',
+      key: 'virtual_ips',
+      className: 'table-col-m',
+      render: (content) => content?.join(', '),
+    },
+    {
+      title: 'Filesystem',
+      key: 'filesystems',
+      className: 'table-col-m',
+      render: (content) => content?.join(', '),
+    },
+    {
+      title: '',
+      key: '',
+      className: 'table-col-xs',
+      render: (_, item) => {
+        const { attributes, resources } = item;
+        return <SiteDetails attributes={attributes} resources={resources} />;
+      },
+    },
+  ],
+};
+
 function AscsErsClusterDetails({
   clusterName,
   cibLastWritten,
   provider,
+  hosts,
   details,
 }) {
+  const [sapSystems, setSapSystems] = useState([]);
   const [currentSapSystem, setSapSystem] = useState(null);
```

Comment:
I know this was already here, however what do you think of streamlining to `setCurrentSapSystem`?
`setSapSystems`/`setSapSystem` look too similar and might cause some headaches.